data/mp/eda_mp_trj.py

```diff
@@ -196,6 +216,7 @@ def tile_count_anno(hist_vals: list[Any]) -> dict[str, Any]:
 
 
 # %% plot histogram of number of sites per element
+# TODO fix the cbar position and weirdness with 6x10e0
```

Comment:
<img width="640" alt="image" src="https://github.com/user-attachments/assets/0f2ab2cd-7c45-4cc0-9a97-2f79ad4250d1">

Reply:
partially addressed in [0d54d68](https://github.com/janosh/matbench-discovery/pull/130/commits/0d54d68a712df0e4e6451715326ef781ef310724)